composer.json

```diff
@@ -50,7 +50,6 @@
         "drupal/chosen": "~2.4",
         "drupal/ckeditor_fixed_toolbar": "dev-1.x#1e4a8191ab289b4c91d9a0f2ac6b40e82e294908",
         "drupal/components": "~1.0",
-        "drupal/config_distro": "^1.0-alpha",
```

Comment:
We are not using this and should remove it.

Reply:
We'd need to keep it so that it can be uninstalled correctly since it's being removed from `core.extension.yml`

Reply:
ah good point. It is installed.